frame/support/Cargo.toml

```diff
@@ -38,6 +38,7 @@ sp-core-hashing-proc-macro = { version = "5.0.0", path = "../../primitives/core/
 k256 = { version = "0.10.2", default-features = false, features = ["ecdsa"] }
 
 [dev-dependencies]
+serde_json = "1.0.79"
```

Comment:
I just realized this, why is it only a dev dependency?

Reply:
because it's used at tests only

Reply:
Ah ok, this is `serde_json` and not `serde` itself.